fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1450,6 +1452,45 @@ private void handleExportStmt(UUID queryId) throws Exception {
         ExportStmt exportStmt = (ExportStmt) parsedStmt;
         exportStmt.setExportStartTime(context.getStartTime());
         context.getGlobalStateMgr().getExportMgr().addExportJob(queryId, exportStmt);
+
+        if (!exportStmt.getSync()) {
+            return;
+        }
+        // poll and check export job state
+        while (true) {
```

Comment:
It is better not to use infinite loop，maybe you can use export_task_default_timeout_second + add a little more time